doc-coverage/raco.rkt

```diff
@@ -0,0 +1,75 @@
+#lang racket
+
+(require raco/command-name
+         racket/cmdline
+         rackunit/docs-complete
+         "main.rkt")
+
+(module+ main
+
+  (define binding (make-parameter #f))
+  (define ratio (make-parameter #f))
+  (define ignore (make-parameter #f))
+  (define error-on-exit? (make-parameter #f))
```

Comment:
I think it would be better to pull out the parameters and any helper functions to the surrounding module, make a function that accepts the files to check, and have the main submodule just contain a `command-line` form that calls the function.